internal/cmd/opensearch/instance/create/create.go

```diff
@@ -0,0 +1,253 @@
+package create
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"stackit/internal/pkg/args"
+	"stackit/internal/pkg/confirm"
+	cliErr "stackit/internal/pkg/errors"
+	"stackit/internal/pkg/examples"
+	"stackit/internal/pkg/flags"
+	"stackit/internal/pkg/globalflags"
+	"stackit/internal/pkg/projectname"
+	"stackit/internal/pkg/services/opensearch/client"
+	opensearchUtils "stackit/internal/pkg/services/opensearch/utils"
+	"stackit/internal/pkg/spinner"
+	"stackit/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/opensearch"
+	"github.com/stackitcloud/stackit-sdk-go/services/opensearch/wait"
+)
+
+const (
+	instanceNameFlag         = "name"
+	enableMonitoringFlag     = "enable-monitoring"
+	graphiteFlag             = "graphite"
+	metricsFrequencyFlag     = "metrics-frequency"
+	metricsPrefixFlag        = "metrics-prefix"
+	monitoringInstanceIdFlag = "monitoring-instance-id"
+	pluginFlag               = "plugin"
+	sgwAclFlag               = "acl"
+	syslogFlag               = "syslog"
+	planIdFlag               = "plan-id"
+	planNameFlag             = "plan-name"
+	versionFlag              = "version"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	PlanName string
+	Version  string
+
+	InstanceName         *string
+	EnableMonitoring     *bool
+	Graphite             *string
+	MetricsFrequency     *int64
+	MetricsPrefix        *string
+	MonitoringInstanceId *string
+	Plugin               *[]string
+	SgwAcl               *[]string
+	Syslog               *[]string
+	PlanId               *string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create an OpenSearch instance",
+		Long:  "Create an OpenSearch instance.",
```

Comment:
Nitpick: Do we have a standard on what help descriptions end with full stop or not

Reply:
The way I see it:
- `Short`: is not a full sentence, it's just a short description of the command that shows in one line for each of the child commands of a given command
- `Long`: might be composed of multiple sentences spanning over multiple lines, so I think it should be a full sentence and end with `.`s

Reply:
Take e.g. `gh pr list`.

- `Short`
<img width="399" alt="image" src="https://github.com/stackitcloud/stackit-cli/assets/28832884/3d26037d-ef51-4af9-8127-3661600e784d">

- `Long`: 
<img width="719" alt="image" src="https://github.com/stackitcloud/stackit-cli/assets/28832884/3dd82fe5-64da-4624-83b8-1fd1e8cdc041">

Reply:
This not applied to the other added commands. They are missing the `.` on the long descriptions

Reply:
I will create a separate PR to make it uniform in all CLI commands